### PR TITLE
Update: Submit button

### PR DESCRIPTION
**Pull request by jzhzhang66**
- Swapped out square button in story inquiry for submit button 
- Deleted submit button folder / component

### CODE REVIEW

**Review by isabellberzsenyi — APPROVED** [review]
marvelous

File: components/contact-page/contact-form/ContactForm.js [hunk 1]
```diff
@@ -80,17 +81,20 @@ export default function ContactForm({ togglePopup }) {
         />
       </InputBoxWrapper>
       <ButtonWrapper>
-        <SquareButton
+        <SubmitButton
```
Comment (josephmannis) [inline components/contact-page/contact-form/ContactForm.js:84]: Can you add `role="button"`?

File: components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.js [hunk 2]
```diff
@@ -379,21 +380,22 @@ export default function StoryInquiryForm({
             })}
           </InputFieldWrapper>
         </ResourceLinksContainer>
-        <SquareButtonWrapper>
-          <SquareButton
-            buttonText="SUBMIT MY STORY"
+        <ButtonWrapper>
+          <SubmitButton
+            aria-label="Submission Button"
+            color={colors.WHITE}
```
Comment (josephmannis) [inline components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.js:385]: Same here, you probably want to add the aria stuff as attrs on the styled component.

**Review by josephmannis — CHANGES_REQUESTED** [review]
One lil thang

**Pull request closed without merging**